broker-daemon/index.js

```diff
@@ -242,6 +247,13 @@ class BrokerDaemon {
   validateEngines () {
     this.engines.forEach((engine, _) => engine.validateEngine())
   }
+
+  async validateExternalAddress () {
+    const externalAddressReachable = await isReachable(this.externalAddress)
+    if (!externalAddressReachable) {
+      this.logger.warning(`External Address {${this.externalAddress}} is not reachable. Broker might not be able to work properly.`)
```

Comment:
`this.logger.warn`

Reply:
Maybe `External Address {${this.externalAddress}} is not reachable from the broker.`